tests/urlpatterns_reverse/tests.py

```diff
@@ -361,6 +361,27 @@ def test_resolver_reverse(self):
         self.assertEqual(resolver.reverse('named-url2', 'arg'), 'extra/arg/')
         self.assertEqual(resolver.reverse('named-url2', extra='arg'), 'extra/arg/')
 
+    def test_resolver_reverse_conflict(self):
+        """
+        Verifies that url names can indeed conflict: they need not be unique.
```

Comment:
The docstring should state the expected behavior rather than including preambles like "Tests that", "Verfies that..." (that is the purpose of all tests).